debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java

```diff
@@ -379,6 +379,70 @@ public void update() throws Exception {
         stopConnector();
     }
 
+    @Test
+    public void transactionMetadata() throws Exception {
+        final int RECORDS_PER_TABLE = 5;
+        final int ID_START = 10;
+        final Configuration config = TestHelper.defaultConfig()
+                .with(SqlServerConnectorConfig.SNAPSHOT_MODE, SnapshotMode.INITIAL)
+                .with(SqlServerConnectorConfig.PROVIDE_TRANSACTION_METADATA, true)
+                .build();
+
+        start(SqlServerConnector.class, config);
+        assertConnectorIsRunning();
+
+        Testing.Print.enable();
+        // Wait for snapshot completion
+        consumeRecordsByTopic(1);
+
+        connection.setAutoCommit(false);
+        final String[] inserts = new String[RECORDS_PER_TABLE * 2];
+        for (int i = 0; i < RECORDS_PER_TABLE; i++) {
+            final int id = ID_START + i;
+            inserts[2 * i] = "INSERT INTO tablea VALUES(" + id + ", 'a')";
+            inserts[2 * i + 1] = "INSERT INTO tableb VALUES(" + id + ", 'b')";
+        }
+        connection.execute(inserts);
+        connection.setAutoCommit(true);
+
+        connection.execute("INSERT INTO tableb VALUES(1000, 'b')");
+
+        // BEGIN, data, END, BEGIN, data
+        final SourceRecords records = consumeRecordsByTopic(1 + RECORDS_PER_TABLE * 2 + 1 + 1 + 1);
+        final List<SourceRecord> tableA = records.recordsForTopic("server1.dbo.tablea");
+        final List<SourceRecord> tableB = records.recordsForTopic("server1.dbo.tableb");
+        final List<SourceRecord> tx = records.recordsForTopic("__debezium.transaction.server1");
```

Comment:
As that's a user-facing topic -- unlike heartbeats -- I'd suggest to stick to the established naming pattern for topics and make it `<logical name>.transactions`, e.g. `dbserver1.transactions`.

Reply:
Done